src/ag-grid-styles.scss

```diff
@@ -0,0 +1,30 @@
+
+@import "./node_modules/@ag-grid-community/core/dist/styles/ag-grid.scss";
```

Comment:
This imports the base styles for ag-grid, and then we override some specific class styles.